sourcegraph/about
# 4.0 Launch 🌌

This closes #5764 which includes all 4.0 related PRs.

### Changelog
- homepage updates
- 4.0 landing page updates
- CtaSection updates with default props; consistent copy, links pointing to signup/demo
- new 4.0 blog posts
- pricing page updates tailored for 4.0

### Test
On the above changelog items:
- check that all links go to their proper locations (ie: all get started CTAs should go to new signup flow)
- check for neat responsiveness and content flow
